app/src/main/java/com/google/android/gms/samples/vision/ocrreader/aadhaar/utils/CardListAdapter.java

```diff
@@ -0,0 +1,106 @@
+package com.google.android.gms.samples.vision.ocrreader.aadhaar.utils;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import com.google.android.gms.samples.vision.ocrreader.R;
+import com.google.android.gms.samples.vision.ocrreader.aadhaar.SavedAadhaarCardActivity;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+/**
+ * Created by yoti on 24/11/2016.
+ */
+
+public class CardListAdapter extends ArrayAdapter<ArrayList> {
```

Comment:
This does not use the view holder pattern ? threfore it does use the RecyclerView ?